source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,134 @@
+=======================
+Time Series Collections
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about **time series collections** in
+MongoDB, and how to interact with them in the MongoDB Go driver.
+
+Time series collections efficiently store sequences of measurements over
+a period of time. The collection consists of time series data with the
+following information:
+
+- Data collected over time
+- Metadata that describes the measurement
+- Date of the measurement
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Example
+     - Measurement
+     - Metadata
+
+   * - Sales Data
+     - Revenue
+     - Company
+
+   * - Infection Rates
+     - Amount of People Infected
+     - Location
+
+Create a Time Series Collection
+-------------------------------
+
+To create a time series collection, pass the following parameters to the
+``CreateCollection()`` function:
+
+- The name of the new collection to create
+- The ``TimeSeriesOptions`` object specifying at least the time field
+
+.. important::
+
+   Versions prior to MongoDB 5.0 cannot create a time series collection.
+
+Example
+~~~~~~~
+
+The following example creates the ``spring_weather.march2022`` time series
+collection the ``temperature`` as the time field:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/timeSeries.go
+   :start-after: begin create ts coll
+   :end-before: end create ts coll
+   :emphasize-lines: 2, 5
+   :language: go
+   :dedent:
+
+To check if you created the collection, send the ``"listCollections"``
+command to the ``RunCommand()`` function:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/timeSeries.go
+   :start-after: begin check ts coll
+   :end-before: end check ts coll
+   :emphasize-lines: 1, 4
+   :language: go
+   :dedent:
+
+After running this example, the output resembles the following:
+
+.. code-block:: json
+   :emphasize-lines: 9, 11, 15
+
+   {  
+      ...
+      "cursor": {
+           "firstBatch": [
+               {
+                   "info": {
+                       "readOnly": false
+                   },
+                   "name": "march2022",
+                   "options": {
+                       "timeseries": {
+                           ...
+                       }
+                   },
+                   "type": "timeseries"
+               },
+       ...
+    }
```

Comment:
issue: This should be converted to an [io-codeblock](https://wiki.corp.mongodb.com/display/DE/How-To%3A+Add+Code+Output+for+Code+Snippets) imo.